examples/va/nvidia-mdev/README.md

```diff
@@ -0,0 +1,165 @@
+data:
+  25-cpu-pinning-nova.conf: |
+    [libvirt]
+    live_migration_completion_timeout = 0
+    live_migration_downtime = 500000
+    live_migration_downtime_steps = 3
+    live_migration_downtime_delay = 3
+    live_migration_permit_post_copy = false
+    [devices]
+    enabled_vgpu_types=nvidia-228,nvidia-229
+    [vgpu_nvidia-228]
+    device_addresses=0000:82:00.0
+    [vgpu_nvidia-229]
+    device_addresses=0000:04:00.0
+kind: ConfigMap
+metadata:
+  name: cpu-pinning-nova
+  namespace: openstack
+```
+
+#### Provider.yaml
+
+In order to easily take advantage of multiple Mdev types in an environment when
+creating flavors, we can associate traits to specific resource providers. With
+provier.yaml we can map those traits and apply them as part of a deployment.
+
+```YAML
+---
+apiVersion: v1
+data:
+  provider.yaml: |
+    meta:
+      schema_version: "1.0"
+    providers:
+      - identification:
+          name: edpm-compute-0.ctlplane.example.com_pci_0000_04_00_0
+        traits:
+          additional:
+            - CUSTOM_NVIDIA_229
+      - identification:
+          name: edpm-compute-0.ctlplane.example.com_pci_0000_82_00_0
+        traits:
+          additional:
+            - CUSTOM_NVIDIA_228
+      - identification:
+          name: edpm-compute-1.ctlplane.example.com_pci_0000_04_00_0
+        traits:
+          additional:
+            - CUSTOM_NVIDIA_229
+      - identification:
+          name: edpm-compute-1.ctlplane.example.com_pci_0000_82_00_0
+        traits:
+          additional:
+            - CUSTOM_NVIDIA_228
+kind: ConfigMap
+  name: compute-provider
+  namespace: openstack
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: compute-provider
+  namespace: openstack
+spec:
+  addCertMounts: false
+  caCerts: combined-ca-bundle
+  dataSources:
+  - configMapRef:
+      name: compute-provider
+  - configMapRef:
+      name: cpu-pinning-nova
+  - configMapRef:
+      name: sriov-nova
+  - secretRef:
+      name: nova-cell1-compute-config
+  - secretRef:
+      name: nova-migration-ssh-key
+  edpmServiceType: nova
+  playbook: osp.edpm.nova
+  tlsCerts:
+    default:
+      contents:
+      - dnsnames
+      - ips
+      issuer: osp-rootca-issuer-internal
+      networks:
+      - ctlplane
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneDeployment
+metadata:
+  name: edpm-deployment-post-driver
+  namespace: openstack
+spec:
+  ansibleExtraVars:
+    edpm_reboot_strategy: force
+  nodeSets:
+  - openstack-edpm
+  preserveJobs: true
+  servicesOverride:
+  - reboot-os
+  - compute-provider
+```
+
+## Stages
+All stages must be executed in the order listed below. Everything is required unless otherwise indicated.
+
+1. [Install the OpenStack K8S operators and their dependencies](../../common/)
+2. [Configuring networking and deploy the OpenStack control plane](control-plane.md)
+3. [Configure and deploy the initial dataplane](edpm-pre.md)
```

Comment:
for lines 163-165, with 3 sentences all pointing to the same link, could we just have:
```
[Update the dataplane](edpm-post.md)
```

Reply:
Yes that makes sense, I'll update.